### PR TITLE
Nicolas Oulianov: solves pandas question

### PR DESCRIPTION
**Pull request by oulianov**
Cheers.

Didn't write docstrings but maybe I will need to

### CODE REVIEW

**Comment by tomMoral** [comment]
Nice one!

**Comment by tomMoral** [comment]
If you are finished, you can sign up for this RAMP challenge: https://ramp.studio/events/brevet_datacamp2020
This is what we are going to do this afternoon. The goal is to try and solve the challenge.
You need to click on the blue button: `Join the event` and I need to approve your participation so send me a message once you have done this.
You can then read the starting kit and go to the left menu to submit your solutions in the sandbox.

File: pandas_questions.py [hunk 1]
```diff
@@ -11,14 +11,23 @@
...
-
+    referendum = pd.read_csv("data/referendum.csv",
+                             sep=';', error_bad_lines=False)
+    regions = pd.read_csv("data/regions.csv")
+    # Departement 01 => Department 1
+    regions['code'] = regions['code'].apply(
+        lambda x: str(int(x)) if x.isdigit() else x)
+    # Drop DOM TOM COM regions.
+    regions = regions[regions['code'].apply(
+        lambda x: x not in ['COM', '1', '2', '3', '4', '5', '6'])]
```
Comment (tomMoral) [inline pandas_questions.py:27]: you can also do this with `isin`.

**Review by tomMoral — APPROVED** [review]
LGTM!

File: pandas_questions.py (hunk 1 above)
Comment (oulianov) [reply pandas_questions.py:27]: Thank you for the hint. If I did this assignment after the course on Pandas I would have changed a lot of things. Glad you are still satisfied !

**Pull request closed without merging**